CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ find_package(Threads REQUIRED)
 find_package(Boost CONFIG REQUIRED
   COMPONENTS
     headers
+    json
```

Comment:
Bad news per your CI run. Boost::Json was introduced in boost 1.75, but our CI env is Jammy based and comes with 1.74: `- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.74.0/BoostConfig.cmake (found suitable version "1.74.0", minimum required is "1.74") found components: headers log log_setup`.

It is really more of a CI problem than a release problem, since our release builds use conan and will pull Boost as needed. But, it is still a problem.